lib/internal/assert/snapshot.js

```diff
@@ -51,7 +51,6 @@ function getTarget() {
 }
 
 function serializeName(name) {
-  validateString(name, 'name');
```

Comment:
Quick question: Is there a specific reason to remove this validation in here?

Reply:
moved it to the callsite